npm run deploy error

Hi,

I´m trying running the deploy script, houwever i receiving the follow message.

`> lambda-cloudwatch-slack@0.3.0 deploy /home/ubuntu/lambda-cloudwatch-slack
> ./scripts/deploy.sh

=> Moving files to temporary directory
Error: Cannot copy '.' to a subdirectory of itself, '/home/ubuntu/lambda-cloudwatch-slack/lambda-cloudwatch-slack-lambda'.
    at checkStats (/home/ubuntu/lambda-cloudwatch-slack/node_modules/fs-extra/lib/copy/copy.js:240:17)
    at fs.stat (/home/ubuntu/lambda-cloudwatch-slack/node_modules/fs-extra/lib/copy/copy.js:227:14)
    at callback (/home/ubuntu/lambda-cloudwatch-slack/node_modules/graceful-fs/polyfills.js:295:20)
    at FSReqWrap.oncomplete (fs.js:153:5)

npm ERR! Linux 4.15.0-1056-aws
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "deploy"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! lambda-cloudwatch-slack@0.3.0 deploy: `./scripts/deploy.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the lambda-cloudwatch-slack@0.3.0 deploy script './scripts/deploy.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the lambda-cloudwatch-slack package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./scripts/deploy.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs lambda-cloudwatch-slack
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls lambda-cloudwatch-slack
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/lambda-cloudwatch-slack/npm-deb
[npm-debug.log](https://github.com/assertible/lambda-cloudwatch-slack/files/4287380/npm-debug.log)

ug.log
`

`ubuntu@api-h-beholder:~/lambda-cloudwatch-slack$ npm -v
3.5.2
ubuntu@api-h-beholder:~/lambda-cloudwatch-slack$
ubuntu@api-h-beholder:~/lambda-cloudwatch-slack$ node -v
v8.10.0
`

--- a/package.json
+++ b/package.json
@@ -1,8 +1,8 @@
 {
   "name": "lambda-cloudwatch-slack",
-  "version": "0.3.0",
+  "version": "0.4.0",
   "description": "Better Slack notifications for AWS CloudWatch",
-  "authors": [    
+  "authors": [
     "Christopher Reichert <creichert07@gmail.com>",
     "Cody Reichert <codyreichert@gmail.com>",
     "Alexandr Promakh <s-promakh@ya.ru>"
@@ -17,7 +17,7 @@
     "url": "^0.11.0"
   },
   "devDependencies": {
-    "node-lambda": "0.12.0"
+    "node-lambda": "0.16.0"
   },
   "keywords": [
     "aws",

